src/features/Chat/components/ActiveWindow/index.tsx

```diff
@@ -12,9 +12,9 @@ import MessageList from './MessageList';
 import { palette } from '@/components/variables';
 
 const ActiveWindow = () => {
-  const [isLoading] = useSendMessageMutation();
-
   const chat = useAppSelector(selectActiveChat);
+
+  const [isLoading] = useSendMessageMutation();
```

Comment:
So this what is called isLoading, is actually the first element of the tuple that is returned from the hook. And that is the trigger to start the mutation. 
See more here: https://redux-toolkit.js.org/rtk-query/usage/mutations#mutation-hook-behavior 

In the variable loading we are asserting that the callback exists?

if we wan to theck if the mutation is loading, you need to use it like this:

```typescript
const [sendMessage, {isLoading}] = useMutation()
```

Then when you use sendMessage, it will keep the isLoading variable in up to date

Reply:
Additionally, I would use it like this:

```typescript
//line 17
const [sendMessage, {isLoading: isMessageSendLoading}] = useMutation()

//line 22
const isLoading = isLoadingMessages || isMessageSendLoading
```


P.S. You also need to call the sendMessage-callback by either passing it as a prop or then bringing some logic to this parent